### PR TITLE
Small performance and correctness improvements

### PR DESCRIPTION
**Pull request by alshopov**

### CODE REVIEW

File: jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ThriftSpanConverter.java [hunk 1]
```diff
@@ -96,9 +96,10 @@
       Map<String, ?> processTags = span.getTracer().tags();
       // add tracer tags to first zipkin span in a process but remove "ip" tag as it is
       // taken care of separately.
-      for (String tagKey : processTags.keySet()) {
-        if (!tagKey.equals(Constants.TRACER_IP_TAG_KEY)) {
-          Object tagValue = processTags.get(tagKey);
+      for (Map.Entry<String, ?> entry : processTags.entrySet()) {
+        String tagKey = entry.getKey();
+        if (!entry.getKey().equals(Constants.TRACER_IP_TAG_KEY)) {
```
Comment (yurishkuro) [inline jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ThriftSpanConverter.java]: `s/entry.getKey()/tagKey/`

Comment (alshopov) [reply jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ThriftSpanConverter.java]: I will update.

**Comment by alshopov** [comment]
> email showed a lot more
Indeed, I had a whole patch series for merge. I got build failure and decided to submit patches one by one. It seems there are problems with building and tests are failing due to timeouts

**Comment by black-adder** [comment]
I landed my change, rebase and the tests should pass.

**Comment by alshopov** [comment]
Well - build failed again. I will regenerate the pull request starting from a branch

**Comment by alshopov** [comment]
Moved to #290

**Pull request closed without merging**